enshrine-esdt-safe/tests/enshrine_esdt_safe_blackbox_test.rs

```diff
@@ -101,34 +150,43 @@ impl EnshrineTestState {
     }
 
     fn propose_setup_contracts(&mut self, is_sovereign_chain: bool) -> &mut Self {
-        self.deploy_enshrine_esdt_contract(is_sovereign_chain);
+        self.deploy_enshrine_esdt_contract(
+            is_sovereign_chain,
+            Some(TokenIdentifier::from(WEGLD_IDENTIFIER)),
+            Some(SOVEREIGN_TOKEN_PREFIX.into()),
+        );
         self.deploy_header_verifier_contract();
         self.propose_set_header_verifier_address();
 
         self
     }
 
-    fn propose_execute_operation(&mut self, has_prefix: bool, error_msg: Option<&str>) {
+    fn propose_execute_operation(
+        &mut self,
+        has_prefix: bool,
+        error_status: Option<ErrorStatus>,
+        tokens: &Vec<TestTokenIdentifier>,
+    ) {
         let (tokens, data) = if has_prefix {
-            self.setup_payments(vec![PREFIX_NFT_TOKEN_ID, FUNGIBLE_TOKEN_ID])
+            self.setup_payments(tokens)
```

Comment:
same `self.setup_payments(tokens)` is both on if and else

Reply:
fixed